packages/components/src/Combobox/styles.ts

```diff
@@ -27,7 +27,7 @@ export function getInputSpacingStyles(size?: ComboboxSize) {
 
     case 'md':
     default:
-      return tw`pl-9`;
+      return tw`pl-10 text-base`;
```

Comment:
I think leaving the padding at 9 should be fine
```suggestion
      return tw`pl-9 text-base`;
```

![Screen Shot 2022-11-14 at 3 20 52 pm](https://user-images.githubusercontent.com/72959522/201574797-18f83571-1738-4da2-9cf0-f51d0bca8724.png)